src/pe.rs

```diff
@@ -1504,6 +1510,18 @@ pub const IMAGE_REL_EBC_SECTION: u16 = 0x0003;
 /// Offset within section
 pub const IMAGE_REL_EBC_SECREL: u16 = 0x0004;
 
+//
+// RISCV relocation types
+//
+/// No relocation required
+pub const IMAGE_REL_RISCV_ABSOLUTE: u16 = 0x0000;
+/// 20-bit HI
+pub const IMAGE_REL_RISCV_HI20: u16 = 0x0005;
+/// 12-bit LOW I
+pub const IMAGE_REL_RISCV_LOW12I: u16 = 0x0007;
+/// 12-bit LOW S
+pub const IMAGE_REL_RISCV_LOW12S: u16 = 0x0008;
+
```

Comment:
Please use the Microsoft names for these, and move them to the same place as the other based locations.